File: CS/GridControlWithBar/PropertyGridControlDescendant/CustomButtonsEventArgs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GridControlWithBar
{
    public class CustomButtonsEventArgs : EventArgs
    {
        public string Name { get; set; }
        public CustomButtonsEventArgs(string _name)
        {
            Name = _name;
        }
    }
}

File: VB/GridControlWithBar/PropertyGridControlDescendant/CustomButtonsEventArgs.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace GridControlWithBar
	Public Class CustomButtonsEventArgs
		Inherits EventArgs
		Private privateName As String
		Public Property Name() As String
			Get
				Return privateName
			End Get
			Set(ByVal value As String)
				privateName = value
			End Set
		End Property
		Public Sub New(ByVal _name As String)
			Name = _name
		End Sub
	End Class
End Namespace
